src/meta/processors/job/JobManager.cpp

```diff
@@ -66,6 +66,16 @@ JobManager::~JobManager() {
   shutDown();
 }
 
+bool JobManager::spaceExist(GraphSpaceID spaceId) {
+  auto spaceKey = MetaKeyUtils::spaceKey(spaceId);
+  std::string val;
+  auto retCode = kvStore_->get(kDefaultSpaceId, kDefaultPartId, spaceKey, &val);
+  if (retCode == nebula::cpp2::ErrorCode::E_KEY_NOT_FOUND) {
+    return false;
+  }
+  return true;
```

Comment:
This judge looks dangerous, but since true do nothing, I think it's okay.

Reply:
OK, I'll make it explicit.

Reply:
> 

I'm okay with current code. If you modify, then better is better ovo

Reply:
I've changed it, but forget to push it before it was merged..

Reply:
> 

Don't worry, I don't think it must be changed.